muikku/src/main/webapp/resources/scripts/gui/websocket.js

```diff
@@ -119,6 +125,26 @@
       return null;
     },
     
+    _startPinging: function () {
+      this._pingHandle = setInterval($.proxy(function() {
```

Comment:
Usage of setInterval instead of setTimeout could potentially lead to concurrency problems